src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkUniIslService.java

```diff
@@ -77,10 +80,12 @@ public void uniIslDiscovery(Endpoint endpoint, IslInfoData speakerDiscoveryEvent
 
         if (!effectiveReference.isSelfLoop()) {
             carrier.notifyIslUp(endpoint, effectiveReference, islData);
+            sendSlowDiscoveryDisableRequest(endpoint);
         } else {
             log.error("Self looped ISL discovery received: {}", effectiveReference);
         }
         endpointData.put(endpoint, effectiveReference);
+        slowDiscoveryEnabled.putIfAbsent(endpoint, false);
```

Comment:
... why direct set, not via `sendSlowDiscoveryDisableRequest(endpoint)` wrapper?

Reply:
Fixed